docs/docs/cmd/planner/task/task-get.md

```diff
@@ -0,0 +1,24 @@
+# planner task get
+
+Retrieve the details of the specified planner task
+
+## Usage
+
+```sh
+m365 planner task get [options]
+```
+
+## Options
+
+`--taskId <taskId>`
```

Comment:
Following the naming convention, let's not repeat `task` and shorten it to just `id`